CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ Change Log
 
 ### Navigator
 
+- new `navigator-testing` artifact to test `NavEventNavigator`, [see docs](https://freeletics.github.io/mad/navigator/testing/)
+- `NavEvent` and other APIs that were marked as visible for testing are now marked as internal
+- compose navigation APIs are not annotated with `ExperimentalMaterialNavigationApi` anymore
```

Comment:
This is already added below, but makes sense to keep it for this section, not in `#StateMachine`